sal_interfaces/MTM2/MTM2_Telemetry.xml

```diff
@@ -272,6 +327,54 @@
     </item>
 </SALTelemetry>
 
+<SALTelemetry>
+    <Subsystem>MTM2</Subsystem>
+    <EFDB_Topic>MTM2_balanceForceData</EFDB_Topic>
```

Comment:
I suggest `forceBalance`... Always put what you are measuring first.  :)

Reply:
I think this refers to the `force balance system`. I am OK either way.

Reply:
This is based on the LTS-162 v5.